docs/stencil-docs/installing-stencil-cli/installing-stencil.md

```diff
@@ -2,22 +2,22 @@
 
 <div class="otp" id="no-index">
 
-### On This Page
+### On this page
 - [Installing on Mac](#installing-on-mac)
 - [Installing on Windows](#installing-on-windows)
 - [Installing on Linux](#installing-on-linux)
-- [Live Previewing a Theme](#live-previewing-a-theme)
+- [Live previewing a theme](#live-previewing-a-theme)
 - [Resources](#resources)
 
 </div>
 
-Stencil CLI gives developers the power to locally edit and preview themes with no impact to a merchant’s live storefront, and its built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts (and set them live) using Stencil CLI's simple, yet powerful commands. 
+Stencil CLI gives developers the power to locally edit and preview themes with no impact to a merchant’s live storefront, and it's built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts (and set them live) using Stencil CLI's simple, yet powerful commands.

```

Comment:
```suggestion
Stencil CLI gives developers the power to locally edit and preview themes with no impact to a merchant’s live storefront, and it's built-in [Browsersync](https://github.com/bigcommerce/browser-sync) capabilities make simultaneous testing across desktop, mobile, and tablet devices a breeze. Once work is complete, developers can push themes to BigCommerce storefronts (and set them live) using Stencil CLI's simple, yet powerful commands.

```
"Its" is referring to Stencil CLI. It should not be it's (or it is) built-in Browswersync...